.gitignore

```diff
@@ -20,4 +20,4 @@
 **yarn-error.log*
 
 **/.idea
-**/.env
\ No newline at end of file
+**/.env.template
```

Comment:
since it's a template i think i can be commited

Reply:
it was discussed that .template file can be used in local to keep the original values so that that can be used to overwrite the .env file when required. and the .env file will be committed with just the keys and not values so that each user can fill them.

Reply:
then maybe its a good idea to have these in readme, and instruct people to make a .env file